aip/0002.md

```diff
@@ -37,16 +38,16 @@ are under discussion.
 
 Common AIPs are organized into the following over-arching categories:
 
-- **A.** Meta-AIPs (AIPs about AIPs)
-- **B.** Resource design
-- **C.** Standard methods
-- **D.** Custom methods
-- **E.** Fields
-- **F.** Design Patterns
-- **G.** Compatibility
-- **H.** File Structure
-- **I.** Annotations
-- **J.** Process
+- **M:** Meta-AIPs (AIPs about AIPs)
+- **RD:** Resource design
```

Comment:
R* ?

Reply:
What is this a reference to?

Reply:
```suggestion
- **R:** Resource design
```

Reply:
Oh I see. I could go either way, but very slightly prefer RD.

Reply:
Very soft opinion: I kind of like the idea of using single chars for the core AIP categories. We have the luxury of grabbing them first. They are also a quick indicator to anybody looking at them that they are part of a core category, and not a custom one added by an adopter.

There are obviously limitations to this idea...

Reply:
@rhamiltonsf I had the same initial thought, but ran into the obvious limitation. I was thinking we reserve both one and two letter prefixes (like Apple does in Objective-C) and say that anything one or two is core.